docs/02-admin/01-installation/docker.md

```diff
@@ -248,3 +291,21 @@ server {
     }
 }
 ```
+
+## Migration Guide
+
+For admins using docker already, as of 2024-08-19, the docker configuration has changed. Read this guide to be sure
+you're up to date. The major change is where media files are stored.
+
+Previously media files were stored at `docker/storage/media`. They will now be stored in `docker/storage/mbin/public/media`.
+The easiest way to migrate them is by running these commands as root (`sudo`)
+
+```shell
+mkdir -p docker/storage/www/public
+# Copy to have the previous files as a backup
+cp -r docker/storage/media docker/storage/www/public
+```
+
+You should then be set all set.
+
+[docker compose merging]: https://docs.docker.com/compose/compose-file/13-merge/
```

Comment:
What is this line? This is invalid markdown syntax.

Reply:
This is actually valid. It's called a "[link reference definition][lrd]" (see the source of this comment for an example). It allows having all your links at the bottom of the document and reusing links.


[lrd]: https://spec.commonmark.org/0.31.2/#link-reference-definition

Reply:
ow like that, yes indeed that is valid.